scientific_library/tvb/simulator/monitors.py

```diff
@@ -216,21 +216,24 @@ class SpatialAverage(Monitor):
 
     """
     _ui_name = "Spatial average with temporal sub-sample"
+    CORTICAL = "cortical"
+    HEMISPHERES = "hemispheres"
 
     spatial_mask = NArray(  #TODO: Check it's a vector of length Nodes (like region mapping for surface)
         dtype=int,
-        label="An index mask of nodes into areas",
-        required=False,
+        label="Spatial Mask",
+        required=HEMISPHERES,
```

Comment:
`required` should be a boolean. I think you meant to change the default on the next field

Reply:
Yes, sorry.